src/functions/plugins.ts

```
@@ -1,19 +1,19 @@
 import { pathToFileURL, fileURLToPath } from 'url';
 import { join, extname, dirname } from 'path';
 import { readdir } from 'fs/promises';
-import { logger } from './client.mts';
+import { logger } from '../utils/index.ts';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
-export async function loadPlugins(): Promise<void> {
-  const pluginsDir = join(__dirname, '../commands');
+export async function syncPlugins(plugin: string, extensions: string = '.ts'): Promise<void> {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)